tests/declare/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -38,7 +39,6 @@ $test = 1;
 declare(ticks=1) {
     $test = 1;
 }
-declare(encoding='ISO-8859-1');
```

Comment:
Let's don't remove this, changing existing tests are bad

Reply:
Reverted the change of ordering.